### ISSUE TITLE
SLA Delayed by Status

### ISSUE THREAD

**Issue by bwhatley87** [osTicket/osTicket#3034]
I looked, but didn't see this... There should be a checkbox on status properties that "pauses" SLA if this status is chosen.

If a ticket is placed on-hold or "waiting for parts" it shouldn't be flagged overdue.

**Comment by LeDistordu** [comment]
+1

**Comment by sktpnk605** [comment]
Yeah, this definitely needs to be a thing. SLA should be purely status based and not state based seeing as how you can have a "Closed" status that would also bet set to a "Closed" state. If this can't be achieved, then there at least needs to be an ability to add a "Waiting" state where the status can use that instead of just "open" or "closed"...definitely a good idea.

**Comment by gaursh** [comment]
+1
Also, what if there is client holidays, over weekends and after office timing, technically SLA should work during the Office hrs only, due to this my most of the tickets are showing Overdue, which is very critical me to handle. 

Once Set any SLA with these filter, All open ticket with this custom SLA PLAN should follow that set time Limit.

**Comment by justfor-it** [comment]
+1 Should be a nice update

**Comment by tapsiturtle** [comment]
+1 Would be a great update
